src/actions/patient.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+import { param } from 'jquery';
+
+
+export function fetchPatients(params= {}) {
+  let queryParams = {_count: 8, _offset:0, '_sort:asc': 'family',...params};
```

Comment:
My Canadian heritage demands a metric-minded approach, and we like our multiples of 10. 👍